plutus-playground/deployment/nixops/default.nix

```diff
@@ -0,0 +1,13 @@
+let
+  hostPkgs = import <nixpkgs> {};
```

Comment:
Use the pinned nixpkgs always

Reply:
I don't think it's really necessary here, but ok